src/bio/SConscript

```diff
@@ -14,8 +14,24 @@ def scons():
     SConscript('smd/SConscript')
     denv.AppendUnique(LIBPATH=['smd'])
 
-    bio = daos_build.library(denv, "bio", Glob('*.c'),
-                             LIBS=['numa', 'spdk', 'smd'])
+    # Link to DPDK static libs
```

Comment:
Does this mean we don't need a DPDK shared lib any more?

Are the SPDK team intending to maintain a private DPDK lib forever now or do they plan to upstream their changes to DPDK and consume an official DPDK release at some point?

Reply:
Because SPDK doesn't build DPDK shared lib anymore (only static libs produced now), they are working on patches for building DPDK shared lib, but the patches haven't been landed yet.

SPDK team will always maintain a submodule DPDK build, at the same time, user may choose not using the it but use an external DPDK build (with matched version). I think DAOS can just use the DPDK from submodule to avoid complications.